### PR TITLE
Fix productization duplicate classes

### PR DESCRIPTION
**Pull request by manstis**
Corollary of https://github.com/kiegroup/kie-wb-distributions/pull/491 for master.

### CODE REVIEW

**Comment by manstis** [comment]
@mbiarnes Could you please approve?

File: kie-drools-wb-parent/kie-drools-wb-distribution-wars/pom.xml [hunk 1]
```diff
@@ -364,7 +364,7 @@
               <finalName>kie-drools-wb-${project.version}</finalName>
               <descriptors>
                 <descriptor>${project.basedir}/src/main/productized/assembly-kie-drools-wb-eap-7-redhat.xml</descriptor>
-                <descriptor>${project.basedir}src/main/productized/assembly-kie-drools-wb-tomcat-8-redhat.xml</descriptor>
+                <descriptor>${project.basedir}/src/main/productized/assembly-kie-drools-wb-tomcat-8-redhat.xml</descriptor>
```
Comment (mbiarnes) [inline kie-drools-wb-parent/kie-drools-wb-distribution-wars/pom.xml]: @manstis this line was removed and then brought back?

Comment (manstis) [inline kie-drools-wb-parent/kie-drools-wb-distribution-wars/pom.xml]: @mbiarnes Look closely... it had a ```/``` added before ```src```

Comment (mbiarnes) [inline kie-drools-wb-parent/kie-drools-wb-distribution-wars/pom.xml]: @manstis closely looked :) +1

**Pull request merged**